source/third_party/eos_portable_archive/portable_iarchive.hpp

```diff
@@ -114,13 +114,16 @@
 #elif BOOST_VERSION < 104800
 #include <boost/spirit/home/support/detail/integer/endian.hpp>
 #include <boost/spirit/home/support/detail/math/fpclassify.hpp>
-#else
+#elif BOOST_VERSION < 106900
 #include <boost/spirit/home/support/detail/endian/endian.hpp>
 #include <boost/spirit/home/support/detail/math/fpclassify.hpp>
+#else
+#define BOOST_MATH_DISABLE_STD_FPCLASSIFY
```

Comment:
Do you still need this??? since it's defined here: https://github.com/CanalTP/navitia/pull/3289/files#diff-c3f0d33993995f2b0e607af4cf92fb07R47

Reply:
this line is deleted:
#define BOOST_MATH_DISABLE_STD_FPCLASSIFY